db/migrate/20230321232519_create_businesses_hiring_invoice_requests.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBusinessesHiringInvoiceRequests < ActiveRecord::Migration[7.0]
+  def change
+    create_table :businesses_hiring_invoice_requests do |t|
+      t.belongs_to :business, foreign_key: true
+      t.text :billing_address
+      t.string :developer_name
+      t.string :position
+      t.date :start_date
+      t.integer :annual_salary
+      t.integer :employment_type
+      t.text :feedback
```

Comment:
Every column that gets validated for presence should have a `null: false` added here.

Reply:
fixed in ae362edf4025f53688e0d722ab91204646b0d42b